src/components/Organization/Dashboard/ProcessCard.tsx

```diff
@@ -11,13 +16,76 @@ const ProcessCard = () => {
   if (!election) return null
 
   return (
-    <Link as={RouterLink} to={`/processes/${ensure0x(election.id)}`} w='full'>
+    <Link as={RouterLink} to={`/processes/${ensure0x(election.id)}`} w='full' textDecoration='none'>
       <ContentsBox w='full'>
-        <ElectionTitle fontSize='md' textAlign='left' />
-        <ElectionSchedule fontSize='xs' textAlign='left' />
+        <Flex
+          flexDirection={{ base: 'column', lg: 'row' }}
+          alignItems={{ base: 'start', lg: 'center' }}
+          justifyContent='space-between'
+        >
+          <ElectionStatusBadge fontSize='text-sm' mb={{ base: 3, lg: 0 }} order={{ lg: '2' }} />
+          <ElectionTitle fontSize='text' textAlign='left' fontWeight='extrabold' order={{ lg: '1' }} />
+        </Flex>
+        <HStack mb={3}>
+          <MdHowToVote />
+          <Text fontWeight='600' fontSize='text-sm' color='dashboard_card_text' className='dashborad_align_card_text'>
+            {(election as any).voteCount} ({((election as any).voteCount / (election as any).census.size) * 100}%) votes
```

Comment:
Why are you using `as any` here? And also, round the result, i.e. `Math.round(value / 100) * 100`